packaging/conda/README.md

```diff
@@ -7,6 +7,9 @@ You'll need `conda-build` installed in your system.
 # At the root of the repo
 # These commands will leave a ready to use conda channel in `./khiops-conda`
 
+# khiops version must be set before launching the build (can be retrieved with the script scripts/khiops-version)
```

Comment:
We also need to eliminate dashes from prerelease versions (see relevant stage in conda.yml https://github.com/KhiopsML/khiops/blob/9a9b80743c57a3594f6cec8fa6f51177c19d54c3/.github/workflows/conda.yml#L59). This needs to be documented IMHO.